cloudsupport/v1/ekssupport.go

```diff
@@ -107,3 +136,24 @@ func (eksSupport *EKSSupport) GetContextName(cluster string) string {
 	}
 	return ""
 }
+
+// Getcfgmap returns the authdata containing mappings of iam-roles/groups or iam-users/groups
+func (EKSSupport *EKSSupport) GetAwsCfgMap(kapi *k8sinterface.KubernetesApi, namespace string) error {
```

Comment:
The functions should return an EKS configMap

```suggestion
func (EKSSupport *EKSSupport) GetEKSCfgMap(kapi *k8sinterface.KubernetesApi, namespace string) (*EKSCfgMap ,error) {
```

Reply:
@dwertent ,can i use already available v1.configMap or should i create a new struct `EKSCfgMap`with same fields ?